templates/cli/lib/commands/init.js.twig

```diff
@@ -23,7 +23,7 @@ const { accountGet } = require("./account");
 const { loginCommand } = require("./generic");
 const { sdkForConsole } = require("../sdks");
 
-const initProject = async () => {
+const initProject = async ({ organizationId, projectId, projectName } = {}) => {
```

Comment:
We are no longer setting project name in this method, I believe. It's a separate method `appwrite pull proejct`

Reply:
Addressed